### ISSUE TITLE
CHORE: Rephrase src/customErrors.ts:25 to: `Action not allowed. addIfNotFound and replaceVal as a function isn't allowed at the same time.`

### ISSUE THREAD

**Issue by objmj-itminds** [IT-Minds-opensource/ts-array-ext#10]
Maybe rephrase src/customErrors.ts  to: `Action not allowed. addIfNotFound and replaceVal as a function isn't allowed at the same time.`
That would make it clearer in my opinion.

_Originally posted by @phillip-itminds in https://github.com/IT-Minds-opensource/ts-array-ext/pull/7#r503742043_

**Comment by IpsumCapra** [comment]
I'll give it a try, this is till open, right? You can hopefully expect a PR soon.

**Comment by objmj-itminds** [comment]
> 
> 
> I'll give it a try, this is till open, right? You can hopefully expect a PR soon.

Yes, it is open :) we are new to creating opensource libraries here at IT Minds, so excuse us if the issue wasn't the most detailed.
